hadoop-shaded-protobuf37/pom.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Let's use the license text here instead:
https://www.apache.org/legal/src-headers.html#headers
This is for the code submitted to ASF projects

Reply:
Thanks @jojochuang for pointing out.
I will update it soon.
I copied from https://github.com/apache/hadoop/blob/eeb58a07e24e6a1abdf32e1c198a5a1e9c2a8f1a/pom.xml#L2. 
I think most of hadoop's pom.xml needs to be updated in that case. :)